### DISCUSSION TITLE
其实我就想知道经济啥时候崩 我好准备回农村养猪

### DISCUSSION THREAD

**Discussion by remixzyb** [WeNeedHome/SummaryOfLoanSuspension#335] [Ideas]
现在的我对gj无任何感觉 被如此高压的我只想说一句 tmd都去毁灭把

**Comment by zhihuacui** [comment]
经济崩溃的时候，没有一只猪猪是无辜的

**Comment by blacklion-spec** [comment]
这片土地上的人民享受过一天自由和民主吗？

**Comment by GitHeP** [comment]
别急我估计快了

**Comment by mehhss** [comment]
怎么连国家两个字都不敢打出来，这里不是支国控制的范围内

**Reply by miaodi** [reply]
崩了以后回去搞团练，当个当地小霸王岂不美哉。

**Reply by zhaomingtao-fighting** [reply]
> 崩了以后回去搞团练，当个当地小霸王岂不美哉。

没准成了开国元勋了

**Comment by oceanzhang860411** [comment]
想多啦，真崩潰了，農村更是地獄，鄉長魚肉百姓的事，還沒看夠嗎？

**Reply by zhaokai5520** [reply]
回去至少有碗饭吃

**Reply by oceanzhang860411** [reply]
建議參考大吃飽時代，不要抱太高期望

**Reply by iosyyy** [reply]
> 建議參考大吃飽時代，不要抱太高期望

乃时候是生产力太低，和现在不可同日而语。

**Reply by zhw173** [reply]
乡长被活埋的事我也没看够

**Comment by shadowlgd** [comment]
个人散户已经不让养猪了吧，我们那是的，问就是不环保

**Comment by chenjiasu** [comment]
我们安徽这边农村散养猪圈不能占用耕地，你用宅基地养猪一两头可以，十头猪就能臭百米能被别人骂死

**Comment by Franklyn1987** [comment]
新无产阶级都不知道怎么办

**Comment by JusticeEER** [comment]
> 现在的我对gj无任何感觉 被如此高压的我只想说一句 tmd都去毁灭把

都在github了，國家這兩個字都不敢打出來，真的悲哀
